server/service/src/invoice_line/stock_in_line/insert/validate.rs

```diff
@@ -31,6 +31,12 @@ pub fn validate(
         return Err(LocationDoesNotExist);
     }
 
+    if let Some(item_variant_id) = &input.item_variant_id {
+        if !check_item_variant_exists(connection, item_variant_id)? {
```

Comment:
I guess to be thorough we should check that it not only exists but that it's for the right `item_id`?
Not overly worried though...